data-prepper-plugins/opensearch-source/src/main/java/org/opensearch/dataprepper/plugins/source/opensearch/worker/client/SearchAccessorStrategy.java

```diff
@@ -31,7 +32,6 @@ public class SearchAccessorStrategy {
     static final String OPENSEARCH_DISTRIBUTION = "opensearch";
     static final String ELASTICSEARCH_DISTRIBUTION = "elasticsearch";
     static final String ELASTICSEARCH_OSS_BUILD_FLAVOR = "oss";
-    static final String OPENDISTRO_DISTRIUBTION = "opendistro";
```

Comment:
Is there a particular reason to remove this distribution?

Reply:
I removed it because it doesn't seem to exist. I think the distribution will be just like "elasticsearch"

Reply:
It seems that ES 6.x and 7.x could be opendistro. But, ES versions beyond 7.10.2 would be `elasticsearch` only.

Reply:
I don't think elasticsearch or opendistro versions even publish their `distribution`. the root endpoint for elasticsearch does not return a distribution, and I think this was added for opensearch to differentiate it. I will download an opendistro and run it if I can to see what gets returned

Reply:
Confirmed that opendistro does not have a distribution returned, which is same behavior as `elasticsearch`. We are just hard-coding the distribution to be `elasticsearch`. 

```
curl -XGET https://localhost:9200 -u 'admin:admin' --insecure' --insecure
{
  "name" : "22654e359636",
  "cluster_name" : "docker-cluster",
  "cluster_uuid" : "rGtPyarcQ5-tJHR6TsSxJw",
  "version" : {
    "number" : "7.10.2",
    "build_flavor" : "oss",
    "build_type" : "tar",
    "build_hash" : "747e1cc71def077253878a59143c1f785afa92b9",
    "build_date" : "2021-01-13T00:42:12.435326Z",
    "build_snapshot" : false,
    "lucene_version" : "8.7.0",
    "minimum_wire_compatibility_version" : "6.8.0",
    "minimum_index_compatibility_version" : "6.0.0-beta1"
  },
  "tagline" : "You Know, for Search"
}
```